clients/restclient/rest_client.go

```diff
@@ -18,13 +22,16 @@ type RestClient struct {
 
 // NewRestClient creates a new Rest client
 func NewRestClient(host, org, space string, rt http.RoundTripper, jar http.CookieJar, tokenFactory baseclient.TokenFactory) RestClientOperations {
-	t := baseclient.NewHTTPTransport(host, getRestURL(org, space), getRestURL(baseclient.EncodeArg(org), baseclient.EncodeArg(space)), rt, jar)
+	t := baseclient.NewHTTPTransport(host, restBaseURL, restBaseURL, rt, jar)
+
 	client := New(t, strfmt.Default)
 	return RestClient{baseclient.BaseClient{tokenFactory}, client}
 }
 
-func getRestURL(org, space string) string {
-	return "rest/" + org + "/" + space
+func NewManagementRestClient(host string, rt http.RoundTripper, jar http.CookieJar, tokenFactory baseclient.TokenFactory) RestClientOperations {
+	t := baseclient.NewHTTPTransport(host, csrfRestBaseURL, csrfRestBaseURL, rt, jar)
+	httpRestClient := New(t, strfmt.Default)
+	return &RestClient{baseclient.BaseClient{TokenFactory: tokenFactory}, httpRestClient}
 }
 
 func (c RestClient) PurgeConfiguration(org, space string) error {
```

Comment:
Could we reuse the method executeRestOperation for the PurgeConfiguration method as well?

Reply:
Ok

Reply:
Done